tests/Tests/500__jupyterhub/test-jupyterlab-notebook.robot

```diff
@@ -66,6 +67,7 @@ Verify A Default Image Is Provided And Starts Successfully
     @{user_data} =    Get Previously Selected Notebook Image Details
     Launch JupyterHub Spawner From Dashboard
     Spawn Notebook
+    Wait For JupyterLab Splash Screen  timeout=30
```

Comment:
This change seems unrelated to this PR

Reply:
Yes, it is related to one of the previous PR which was merged but was failing because of not waiting for jupyterlab splash screen